lib/addons/amp/collector-config-amp.ytpl

```diff
@@ -9,7 +9,14 @@ metadata:
   namespace: "{{namespace}}"
 spec:
   mode: "{{deploymentMode}}"
-  image: public.ecr.aws/aws-observability/aws-otel-collector:v0.21.0
+  image: public.ecr.aws/aws-observability/aws-otel-collector:v0.33.1
+  resources:
+    limits:
+      cpu: "1"
+      memory: "2Gi"
```

Comment:
Should not this be configurable? Do you foresee cases that this particular constraint won't work in production for larger volumes?

Reply:
This could be made configurable but this configuration is for collector so we dont run in to constraint in production also this is the config received from service team.

Reply:
I am not questioning the need for the settings, I actually like them. We can apply for now, but these are hard coded and there will be no way for customers to override them if needed. Let's wait for the customer signals ;)